pages/assessment.tsx

```diff
@@ -24,6 +27,7 @@ const Assessment: NextPage = () => {
   const [currentPage, setCurrentPage] = useState<number>(1);
   const [maxItemsPerPage, setMaxItemsPerPage] = useState<number>(4); //max items set to 4 as default
 
+  const taskDescription: string = 'Variabler med nøkkelordet var er globale, mens varibler med nøkkelordet let har et local scope eller blokk scope som vil si at de kun defineres for deler av koden om de defineres inni en kodeblokk.'  
```

Comment:
currently an answer is used as a task description :s

Reply:
Yeah, that'll be fixed in my pr #33 :)